src/components/TimeZoneSelector.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { Autocomplete } from '@terraware/web-components';
+import strings from 'src/strings';
+import { getTimeZones } from 'src/api/timezones/timezones';
+
+export type TimeZoneDescription = {
+  id: string;
+  longName: string;
+};
+
+export type TimeZoneSelectorProps = {
+  onTimeZoneSelected: (tzSelected: TimeZoneDescription) => void;
+  selectedTimeZone?: string;
+  disabled?: boolean;
+};
+
+export default function TimeZoneSelector(props: TimeZoneSelectorProps): JSX.Element {
+  const { onTimeZoneSelected, selectedTimeZone, disabled } = props;
+  const [timeZones, setTimeZones] = useState<TimeZoneDescription[]>([]);
+
+  useEffect(() => {
+    const fetchTimeZones = async () => {
+      const timeZoneResponse = await getTimeZones();
+      if (!timeZoneResponse.error && timeZoneResponse.timeZones) {
+        setTimeZones(
+          timeZoneResponse.timeZones.sort((a, b) => {
+            if (a.longName > b.longName) {
+              return 1;
+            }
+            if (b.longName > a.longName) {
+              return -1;
+            }
+            return 0;
+          })
+        );
+      }
+    };
+
+    fetchTimeZones();
+  }, []);
+
+  const tzToDropdownItem = (tz?: TimeZoneDescription) =>
+    tz ? { label: tz.longName, value: tz.id } : { label: '', value: '' };
+
+  const tzNameToDropdownItem = (tz?: string) => {
+    const foundTimeZone = timeZones.find((iTtimeZone) => iTtimeZone.id.toString() === tz?.toString());
+    return foundTimeZone ? { label: foundTimeZone.longName, value: foundTimeZone.id } : { label: '', value: '' };
+  };
+
+  const tzOptions: any[] = useMemo(() => {
+    return timeZones.map((tz) => tzToDropdownItem(tz));
+  }, [timeZones]);
+
+  const onChangeTimeZone = (timeZone: any) => {
+    const foundTimeZone = timeZones.find((iTtimeZone) => iTtimeZone.id.toString() === timeZone.value?.toString());
+    if (foundTimeZone) {
+      onTimeZoneSelected(foundTimeZone);
+    }
+  };
+
+  const isEqualAut = (optionA: any, optionB: any) => {
```

Comment:
#naming What is "Aut"?

Reply:
Oh, sorry, It came from "autocomplete" because initially I did a <Select> component and an <Autocomplete> component and each one has it owns _isEqual_ function. But finally, I chose the autocomplete an forgot to rename